app/src/main/java/com/mobilecoronatracker/ui/about/CreditsListViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.mobilecoronatracker.ui.about
+
+import android.app.Application
+import androidx.annotation.NonNull
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.MutableLiveData
+import com.mobilecoronatracker.R
+import com.mobilecoronatracker.model.CreditsEntryModel
+import com.mobilecoronatracker.model.CreditsEntryModelImpl
+
+class CreditsListViewModel(@NonNull app: Application) : AndroidViewModel(app), CreditsListViewModelable {
+    override val credits = MutableLiveData<List<CreditsEntryModel>>()
+
+    init {
+        val list = listOf<CreditsEntryModel>(
+            CreditsEntryModelImpl(getString(R.string.icon_header), "made by Smashicons from www.flaticon.com")
+            , CreditsEntryModelImpl(getString(R.string.icon_header), "made by ultimatearm from www.flaticon.com")
```

Comment:
Please move commas to the end of the line. 

Is this line doubled?

Reply:
Commas - ok.
This line is not double, it is for a different author.